### DISCUSSION TITLE
Chapter Support

### DISCUSSION THREAD

**Discussion by Kawsiphek** [pear-devs/pear-desktop#1945] [Ideas]
I listen to many full albums and mixes on youtube. Since the introduction of "chapters" on youtube many artists split up their tracks using the chapters feature, allowing you to scroll through the chapters and see the title of the song on the timeline/seekbar, and also skip ahead to a specific song by picking the chapter.

I think it would be great if the app detected when the song has chapters, and add each song as an entry on the up next track list. Also to  allow you to skip through the chapters using the next and previous buttons. Appending the chapter name to after the title would be a great touch as well, so you can see at a glance the name of the song on the album you are listening to. 

Here is an album as an example: https://www.youtube.com/watch?v=N4VS9EhtZx4
Right now it is just treated as a 1hr 7minute song instead of multiple individual songs.

**Comment by MutatedT7** [comment]
Any updates or suggestions for this? Looking for the same feature :)
